README.md

```diff
@@ -251,6 +252,7 @@ datadogWebpackPlugin({
     auth?: {
         apiKey?: string;
     };
+    customPlugins?: (options: Options, context: GlobalContext) => UnpluginPlugin[];
```

Comment:
I don't think that this type is correct, `customPlugins` expects an array of functions

Reply:
Oh right, I missed a `[]`.
Good catch !